src/main/resources/messages/help_et.yml

```diff
@@ -35,11 +35,11 @@ section:
 # Important: Put main commands like "authme" before their children (e.g. "authme.reload")
 commands:
     authme.register:
-        description: 'Registreeri mängija'
-        detailedDescription: 'Registreeri valitud mängija valitud parooliga.'
+        description: 'Register a player'
```

Comment:
ehm, what?

Reply:
Translated that part again or should I change it back to the English version?